rcon/cli.py

```diff
@@ -423,6 +423,46 @@ def reset_user_settings(server: int):
 
     print("Done")
 
+@cli.command(name="merge_duplicate_player_ids")
+def merge_duplicate_player_ids():
+    logger.info(f"Merging duplicate player ID records")
+    players = {}
+
+    with enter_session() as session:
+        stmt = select(PlayerID)
+        rows = session.execute(stmt).scalars()
+
+        for player in rows:
+            id_, steamid, date = player.id, player.player_id, player.created
```

Comment:
`date` isn't being used? Not an issue on itself but I thought the idea was to preserve oldest?

On that topic, wouldn't it be better to preserve newest, since recent data will be more relevant than old data?